python/examples/quickstarters/ai_productivity/main.py

```diff
@@ -11,15 +12,20 @@
 dotenv.load_dotenv()
 
 # Initialize tools.
-openai_client = OpenAI()
+openai_client = OpenAI(
```

Comment:
Consider adding error handling for missing Helicone API key:
```python
helicone_api_key = os.environ.get('HELICONE_API_KEY')
if not helicone_api_key:
    raise ValueError("HELICONE_API_KEY environment variable is required")
```
This would provide better error messages when the key is missing.